plugins/scaffolder-backend-module-gitlab/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-scaffolder-backend-module-gitlab",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
```

Comment:
```suggestion
    "role": "backend-plugin-module"
```

Reply:
Are you sure about this? When you use `yarn new` and pick `scaffolder-module` (which is where you'd put actions) the result sets the role to `backend-plugin-module`. We use this for our internal actions and it's working fine

Reply:
Sorry, I totally mis-read the diff. :(